kong/db/dao/init.lua

```diff
@@ -710,6 +712,10 @@ local function generate_foreign_key_methods(schema)
         end
 
         local entities, err
+        if options == nil or options.expand_shorthands ~= false then
+          options = options or {}
```

Comment:
Can we set options to an empty array at the top of the function instead of these repeated checks?

Reply:
Actually I think it's impossible since those assignments happen in different methods. Unless I missed something there are no repeated checks within the same context

Reply:
I've moved the options nil check to the top of the corresponding functions.